legal-api/src/legal_api/reports/report.py

```diff
@@ -247,6 +247,7 @@ def _format_address(address):
 
     def _format_incorporation_data(self, filing, report_type):
         filing['header']['reportType'] = report_type
+        filing['header']['filingId'] = self._filing.id
```

Comment:
So filing id is otherwise populated when I save a draft? Will this new code overwrite that?

Reply:
This code will ensure that filing id is always present in the header of the json that gets passed to the report template